src/Symfony/Bundle/SecurityBundle/Resources/config/guard.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Security\Guard\Firewall\GuardAuthenticationListener;
+use Symfony\Component\Security\Guard\GuardAuthenticatorHandler;
+use Symfony\Component\Security\Guard\Provider\GuardAuthenticationProvider;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('security.authentication.guard_handler', GuardAuthenticatorHandler::class)
+            ->call('setSessionAuthenticationStrategy', [service('security.authentication.session_strategy')])
+            ->args([
+                service('security.token_storage'),
+                service('event_dispatcher')->nullOnInvalid(),
+                abstract_arg('stateless firewall keys'),
+            ])
+
+        ->alias('Symfony\Component\Security\Guard\GuardAuthenticatorHandler', 'security.authentication.guard_handler')
+
+        ->set('security.authentication.provider.guard', GuardAuthenticationProvider::class)
+            ->abstract()
+            ->args([
+                abstract_arg('Simple Authenticator'),
```

Comment:
Just `Authenticators` as its iterable and there is no "simple" (yes it already seems wrong in the original xml)

Reply:
Done